templates/reprepro-sign-config.sh.erb

```diff
@@ -0,0 +1,10 @@
+# Configuration variables for reprepro-sign.sh
+GPG_OPTS=""
+
+KEY_OPTS=""
+<% if @signing_key_ids.size != 0 -%>
+<% @signing_key_ids.each do |fingerprint| -%>
+KEY_OPTS="$KEY_OPTS -u <%= fingerprint %>"
+<% end -%>
+<% end -%>
```

Comment:
This could be templated out another way, but I think this is a reasonable pattern.

I realize as I write that the test for a non-zero size is vestigial from when I was planning to make this
```suggestion
<% if @signing_key_ids.size != 0 -%>
KEY_OPTS="$KEY_OPTS -u <%= @signing_key_ids.join(" -u ") %>"
<% end -%>
```

and thus can be removed. Of course, if this version is preferred then the loop can be removed instead.